core/src/test/resources/application.properties

```diff
@@ -11,10 +11,10 @@ spring.liquibase.password=${DATASOURCE_PASSWORD}
 spring.liquibase.enabled=true
 spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
 # Hibernate
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
I just can approve due to this change. Even if it is a developer profile.

In this project only Liquibase is responsible to update the database to certain state.

Reply:
Fix issues with h2 real -> double